net/api/impl/posix_server.h

```diff
@@ -66,6 +67,7 @@ struct Request final {
   const bool has_body;
   const std::string empty_string = "";
   const std::string& body;  // TODO(dkorolev): This is inefficient, but will do.
+  const bricks::time::EPOCH_MILLISECONDS timestamp;
```

Comment:
It's a dedicated type, to prevent users from accidentally adding/subtracting `int`-s or `double`-s to it. Will rename to go away from `UPPER_CASE` ... soon-ish, but not in this PR.